lib/evmone/vm.cpp

```diff
@@ -61,6 +61,15 @@ evmc_set_option_result set_option(evmc_vm* c_vm, char const* c_name, char const*
         vm.add_tracer(create_histogram_tracer(std::clog));
         return EVMC_SET_OPTION_SUCCESS;
     }
+    else if (name == "validate_eof")
```

Comment:
The current convention is to add the option without the `value` (see e.g. `trace` and `histogram`). So the usage is just `evmone.so,validate_eof`.

Reply:
fixed, I used cgoto as a template for a `validate_eof=off` version, then forgot to simplify.